airflow/Dockerfile

```diff
@@ -1,5 +1,7 @@
+
 # syntax=quay.io/astronomer/airflow-extensions:latest
 
-FROM quay.io/astronomer/astro-runtime:9.5.0-base
+FROM quay.io/astronomer/astro-runtime:9.5.0
+#-base
 
-COPY include/airflow_provider_weaviate-0.0.1-py3-none-any.whl /tmp
+# COPY include/airflow_provider_weaviate-0.0.1-py3-none-any.whl /tmp
```

Comment:
Does other part of the DAGs still need it?

Reply:
removed